src/targets/java/okhttp.js

```diff
@@ -58,9 +84,8 @@ module.exports = function (source, options) {
 
   // construct headers
   if (headers.length) {
-    headers.forEach(function (key) {
-      code.push(1, '.addHeader("%s", "%s")', key, source.allHeaders[key])
-    })
+    headers.filter(key => !(source.allHeaders[key].toLowerCase().includes('multipart/form-data'))) // Remove content type header if form-data
+      .forEach((key) => { code.push(1, '.addHeader("%s", "%s")', key, source.allHeaders[key]) })
```

Comment:
Optional suggestion, to make all pre-existing surrounding code here more concise along with your important addition:

instead of **everything that's between `L83` - `L89`** - we can put **if you like**:
```js
_(source.allHeaders)
  .pickBy((_, key) => key.toLowerCase().includes('multipart/form-data'))
  .forEach((value, key) => code.push(1, '.addHeader("%s", "%s")', key, value))
```